deepspeed/launcher/runner.py

```diff
@@ -564,6 +566,7 @@ def main(args=None):
         else:
             cmd = runner.get_cmd(env, active_resources)
 
+    env["PDSH_SSH_ARGS_APPEND"] += f" -p {args.port} "
```

Comment:
I agree w. this suggestion from @loadams. The deepspeed launcher supports different multi-node backends, `pdsh` is the default and most common but it would be good to move the `PDSH_SSH_ARGS_APPEND` variable into the `PDSHRunner` class inside multinode_runner.py.

Reply:
Thank you for your feedback. I've moved this part of the code into the PDSHRunner class as specified, and made the necessary changes.